k8s/mediawiki/schema.yaml

```diff
@@ -69,7 +69,7 @@ properties:
         base64: False
 
   # MySQL related settings
-  # db.mediawikiPassword is the database user's password for the user mediawiki.
+  # db.name is the name of the mediawiki database.
```

Comment:
You may want to check this comment for clarity/accuracy, as I tried my best to fix what seems like an accidentally-duplicated db.mediawikiPassword comment!

Reply:
Yes. You're right!